File: control_de-flujo.py
'''
if
elif
elif
else
'''
#python es identado
x=2
if x==2:
    print("cumple")

if x==3:
    print("primera salida")
elif x=="2":
    print("segunda salida")
elif x==8:
    print("tercera salida")
else:
    print("no encontrado")

#OPERADORES LOGICOS
#or and
if (100>10 and 10==10):
    print("docle condicion ok")
    
if (100>10000 or 10==10):
    print("docle condicion ok")

if True:
    print("siempre es verdadero")

#USO DE MATCH CASE

opcion=8

match opcion:
    case 1:
        print("seleccion 1")
    case 2:
        print("seleccion 2")
    case 3:
        print("seleccion 3")
    case _:
        print("caso por defecto")

File: control_de_flujo.js
/*
if
else if
else
*/
var x=2
if(x==2){
    console.log("cumple")
}
if(x===3){
    console.log("primera salida")
}else if(x==="2"){
    console.log("segunda salida")
}else if(x===2){
    console.log("tercera salida")
}else{
    console.log("no encontrado")
}

//OPERADORES LOGICOS
//&&
//  ||
if (100>10 && 10==10){
    console.log("doble condicion ok")
}

if (100>10000 || 10==10){
    console.log("doble condicion ok")
}
if(true){
    console.log("siempre es verdadero")
}

// USO DE SWITCH CASE

let opcion=2;

switch(opcion){
    case 1:
        console.log("seleccion 1")
        break
      case 2:
        console.log("seleccion 2")
        break
      case 3:
        console.log("seleccion 3")
    default:
        console.log("caso por defecto")
}